tests/ensemble_evaluator/test_ensemble_legacy.py

```diff
@@ -0,0 +1,69 @@
+import threading
+import websockets
+import pytest
+import asyncio
+import logging
+import ert_shared.ensemble_evaluator.entity.identifiers as identifiers
+from pathlib import Path
+from ert_shared.ensemble_evaluator.config import CONFIG_FILE, load_config
+from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
+from ert_shared.ensemble_evaluator.ws_util import wait as wait_for_ws
+from cloudevents.http.event import CloudEvent
+from cloudevents.http import to_json
+
+
+def test_run_legacy_ensemble(tmpdir, unused_tcp_port, make_ensemble_builder):
+    num_reals = 2
+    conf_file = Path(tmpdir / CONFIG_FILE)
+    with tmpdir.as_cwd():
+        with open(conf_file, "w") as f:
+            f.write(f'port: "{unused_tcp_port}"\n')
+
+        ensemble = make_ensemble_builder(tmpdir, num_reals, 2).build()
+        config = load_config(conf_file)
+        evaluator = EnsembleEvaluator(ensemble, config, ee_id="1")
+
+        assert evaluator.get_successful_realizations() == num_reals
+
+
+@pytest.mark.asyncio
+async def test_run_and_cancel_legacy_ensemble(
```

Comment:
The tests sends a termination request that cancels the evaluator :slightly_smiling_face: I have added a couple of comments, does it make more sense now?

Reply:
Thanks, my mistake! With that said I do find some of the tests and occasional parts of the code (especially in the `ee-api`) to be a bit on the imperative side of things. Might be because I'm not speaking `ee`-lingo fluently yet, but intent does sometimes get a bit lost on me at a first read 🤷